### PR TITLE
Remove instructions for Ubuntu 12.04 Trusty, which is no longer supported.

### LINKED ISSUE #2015: Is Ubuntu 12.04 Support Still Needed?
There are various shards of documentation related to Ubuntu 12.04 in the wiki.  Open question: does the Drake community think it's important to continue 12.04 support?  Options are:

- Leave it in the current ambiguous state
- Officially cease support for 12.04 and remove references to it
- Officially support 12.04 with CI coverage
- Your ideas here

### PR DESCRIPTION
**Pull request by david-german-tri**
Resolves #2015

---
This change is [Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/2039) &emsp; Multiple assignees:&emsp;@RussTedrake&nbsp;<a href="/robotlocomotion/drake/pulls/assigned/RussTedrake">RussTedrake</a>,&emsp;@david-german-tri&nbsp;<a href="/robotlocomotion/drake/pulls/assigned/david-german-tri">david-german-tri</a>,&emsp;@jwnimmer-tri&nbsp;<a href="/robotlocomotion/drake/pulls/assigned/jwnimmer-tri">jwnimmer-tri</a>

### CODE REVIEW

File: drake/doc/ubuntu.rst [hunk 1]
```diff
@@ -10,10 +10,6 @@ Install the prerequisites::
     sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
     wget -q -O - http://llvm.org/apt/llvm-snapshot.gpg.key | sudo apt-key add -
 
-    # Ubuntu 12.04 LTS (Precise)
-    sudo add-apt-repository -y "deb http://llvm.org/apt/precise/ llvm-toolchain-precise-3.7 main"
-    sudo add-apt-repository -y ppa:kalakris/cmake
-
     # Ubuntu 14.04 LTS (Trusty)
```
Comment (jwnimmer-tri) [inline drake/doc/ubuntu.rst]: This comment still appears here and on line 26.  I am unclear on what its purpose is.

**Comment by jwnimmer-tri** [comment]
In general, I would also like @RussTedrake to sign off on dropping this platform, before we proceed.

File: drake/doc/ubuntu.rst (hunk 1 above)
Comment (david-german-tri) [inline drake/doc/ubuntu.rst]: Done.

**Comment by jwnimmer-tri** [comment]
LGTM.

**Comment by RussTedrake** [comment]
Can we make the changelog show that we cut this support, and provide a pointer back to this PR.  That way an adventurous 12.04 user could still find his/her way to the old instructions if necessary?

File: drake/doc/ubuntu.rst [hunk 2]
```diff
@@ -10,11 +10,6 @@ Install the prerequisites::
     sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
     wget -q -O - http://llvm.org/apt/llvm-snapshot.gpg.key | sudo apt-key add -
 
-    # Ubuntu 12.04 LTS (Precise)
```
Comment (david-german-tri) [inline drake/doc/ubuntu.rst]: This is a test comment in GitHub to see how it shows up in Reviewable.

File: drake/doc/ubuntu.rst [hunk 3]
```diff
@@ -1,6 +1,6 @@
-***********************************************
-Ubuntu 12.04 LTS (Precise) / 14.04 LTS (Trusty)
-***********************************************
+*************************
+Ubuntu 14.04 LTS (Trusty)
+*************************
```
Comment (david-german-tri) [inline drake/doc/ubuntu.rst]: Another test comment for Reviewable.

**Comment by david-german-tri** [comment]
Added to Changelog.

**Comment by jwnimmer-tri** [comment]
*[CHANGELOG.md, line 32 \[r3\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2039#-KF4zq-mDcp4Rtwu9Xso:-KF4zq-mDcp4Rtwu9Xsp:579245831) ([raw file](https://github.com/robotlocomotion/drake/blob/9cbbda912b3a1665b1227962f0f3b5b47c7cac6c/CHANGELOG.md#L32)):*
Past tense in commit messages and changelogs is a pet peeve of mine.  I am not sure I want to inflict that opinion on Drake, but perhaps consider whether you want to stick with present or subjunctive tense in this case?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2039)*

**Comment by david-german-tri** [comment]
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---

*[CHANGELOG.md, line 32 \[r3\]](https://reviewable.io:443/reviews/robotlocomotion/drake/2039#-KF4zq-mDcp4Rtwu9Xso:-KF50InbBN-Lq-C8krLh:-205228908) ([raw file](https://github.com/robotlocomotion/drake/blob/9cbbda912b3a1665b1227962f0f3b5b47c7cac6c/CHANGELOG.md#L32)):*
Why not.  Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2039)*

**Comment by jwnimmer-tri** [comment]
Reviewed 1 of 1 files at r2, 1 of 1 files at r4.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2039)*

**Comment by jwnimmer-tri** [comment]
:lgtm:

---







*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2039#-:-KF5DMVwBZaJsLT64rsx:1417088721)*

**Comment by david-german-tri** [comment]
+@jwnimmer-tri +@russtedrake 

Hey look, Reviewable supports multiple assignees with this syntax.

---

Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2039#-:-KF5J7SzOS6r-aYJLBJK:1935488011)*

**Comment by david-german-tri** [comment]
@jamiesnape, looks like there was an infrastructure flake in the Mac CI: "Slave went offline during the build"

https://drake-jenkins.csail.mit.edu/job/experimental/758/compiler=clang,label=mac/console

---

Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2039#-:-KF5O0t2Ky5SM3G3Aosp:1301265797)*

**Comment by david-german-tri** [comment]
I'm going to merge despite the flake.

---

Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/2039#-:-KF5OAwTGP9VmBmzX4ly:2025990909)*

**Pull request merged**